### ISSUE TITLE
NetworkTables Client Entry does not update.

### ISSUE THREAD

**Issue by Codadillo** [first-rust-competition/nt-rs#4]
I'm having issues getting the networktables client to function.
With the following code, the entry "test" only gets set to 2.
```rust
let inst = NetworkTables::connect("10.49.4.2:1735", "cool client").await?;
let entry_id = inst.create_entry(EntryData::new("test".to_owned(), 0, EntryValue::Double(1.))).await;
inst.get_entry(entry_id).set_value(EntryValue::Double(2.));
inst.get_entry(entry_id).set_value(EntryValue::Double(3.));
loop {}
```
I've tested this in some other similar cases, and I get the same result where the second set of an entry doesn't work.

**Comment by wozeparrot** [comment]
What version of `nt` are you using?

**Comment by Codadillo** [comment]
I'm using nt = 2.1.0

**Comment by wozeparrot** [comment]
Yea I ran into this as well, same version don't know the cause yet.

**Comment by Redrield** [comment]
Fix published as nt `2.1.1`

**Comment by Codadillo** [comment]
Thanks for the fix!
